### DISCUSSION TITLE
List types always resolve to null after v8 update

### DISCUSSION THREAD

**Discussion by DomeBoi** [graphql-dotnet/graphql-dotnet#4091] [Q&A]
We are trying to migrate from v7 to v8 and are now encountering a very strange problem.

The issue is that all fields using `ListGraphType` are now returning null instead of the expected list of values. This only happened after updating the packages to the latest version.

Even with a simple Query like this:
```C#
Field<ListGraphType<StringGraphType>>("test")
                .Description("Just a test")
                .Resolve(_ => new List<string> { "test1", "test2", "test3" });
```

I now just receive null in the response:
![image](https://github.com/user-attachments/assets/c902ca98-7068-442b-bc16-ae3a07a8aafe)

If anyone has an idea what could cause this I would really appreciate any help!

**Comment by Shane32** [comment]
I've converted this to an issue:
- #4092
